src/docs/custom-mail/index.md

```diff
@@ -0,0 +1,65 @@
+# How to Create a Custom Email
+
+This guide is designed for beginners. If you're already familiar with some basics, you might be able to skip certain pre-requisites and steps.
+
+### Pre-requisites
+Before you begin, make sure you have the following:
+- **Discord Account**: For accessing our official bot and community.
+- **Gmail Account**: For receiving forwarded emails.
+- **ImprovMX Account**: For email forwarding services.
+- **Cloudflare Account**: For managing DNS records.
+- **is-a.dev Domain Name**: Your custom domain for email.
+
+# Step One: Set Up Your Domain
+
+1. **Create Your Custom Domain**:
+   - **Access our Discord**: Join our official [Discord server](https://discord.com/invite/is-a-dev-830872854677422150) if you haven’t already. You will need this to interact with our official bot.
+   - **Run a Command**: Use the provided command in the Discord bot to create your custom domain. This process will guide you through the setup.
+
+2. **Set Up Cloudflare**:
+   - **Sign Up or Log In**: Go to [Cloudflare](https://www.cloudflare.com/) and create an account or log in if you already have one.
+   - **Add Your Domain**: Follow Cloudflare’s prompts to add your newly created domain. Cloudflare will manage your domain’s DNS settings.
+
+   For detailed instructions on setting up Cloudflare and your domain, refer to this [guide](https://www.is-a.dev/docs/cloudflare-pages/).
```

Comment:
The link you gave is only for Cloudflare Pages (which are websites), not for Cloudflare DNS.